sources/packages/forms/src/form-definitions/supportingusersparent2022-2023.json

```diff
@@ -2715,7 +2715,7 @@
                     "strictDateValidation": false,
                     "multiple": false,
                     "unique": false,
```

Comment:
There is a radio button at line 2067.

Reply:
Line 2059 already has
"onlyAvailableItems": true,
which should apply to line 2067.